retrieval/target.go

```diff
@@ -359,9 +364,15 @@ func (t *Target) scrape(sampleAppender storage.SampleAppender) (err error) {
 	}()
 
 	for samples := range t.ingestedSamples {
-		for _, s := range samples {
-			s.Metric.MergeFromLabelSet(baseLabels, clientmodel.ExporterLabelPrefix)
-			sampleAppender.Append(s)
+		if t.proxy {
+			for _, s := range samples {
+				sampleAppender.Append(s)
```

Comment:
Yes, this needs some work to cover the PGW as well, i.e.  attaching `instance` (and others?) if not set but don't attach `exporter_*`.